File: app/models/aluno.js
const Sequelize = require("sequelize");
const _sequelize = new Sequelize({
    dialect: 'sqlite',
    storage: 'c:\\sqlite\\escolar.db'
});

const aluno = _sequelize.define('aluno', {
    id: {
        type: Sequelize.INTEGER,
        autoIncrement: true,
        allowNull: false,
        primaryKey: true
    },
    nome: {
        type: Sequelize.STRING,
        allowNull: false
    },
    sobrenome: {
        type: Sequelize.STRING
    }
})

module.exports = aluno;

File: Escolar/Models/Aluno.cs

using System;

namespace Escolar.Models
{
  public class Aluno
  {
    private int _id;

    public int Id
    {
      get { return _id; }
      set { _id = value; }
    }

    private string _nome;

    public string Nome
    {
      get { return _nome; }
      set { _nome = value; }
    }


    //public int Id { get; set; }
    //public string Nome { get; set; }
    //public string Sobrenome { get; set; }
    //public int TurmaId { get; set; }
  }
}
